docs/design/002-containerized-gadgets.md

```diff
@@ -492,17 +504,26 @@ func main() {
 	myeBPFObject := io.ReadAll()
 	// Or from a go:embed variable
 
-	opts := runner.Opts{
-		// Authentication options
-		// Gadget parameters
-		// etc.
+  r, err := runner.NewRunnerFromBytes(myeBPFObject,
```

Comment:
We would need to be able to pass the wasm and metadata bytes too.

Reply:
And then we probably would need a slice of wasm layers in order to support multiple wasm "enrichers". And ordering.
I think that could be discussed in a follow up PR. WDYT?

Reply:
Yes, this can be in another PR. For now, it could be a `// TODO` comment.

I think one wasm module is enough because that's what we have in images, and one wasm module can contain several enrichers.

Reply:
> I think one wasm module is enough because that's what we have in images, and one wasm module can contain several enrichers.

I thought the following example: I want to use that gadget `xyz` which also comes with a wasm module. And on top of that I want to add my own wasm enricher -> 2 wasm modulels.
Of course have currently have only one layer in images, but just thinking about the future use cases